components/raftstore/src/store/memory.rs

```diff
@@ -51,3 +54,16 @@ pub struct ApplyMemoryTrace {
     pub pending_cmds: usize,
     pub rest: usize,
 }
+
+pub fn needs_evict_entry_cache() -> bool {
+    fail_point!("needs_evict_entry_cache", |_| true);
+    if memory_usage_reaches_high_water() {
+        let usage = get_global_memory_usage();
+        let ec_usage = MEMTRACE_ENTRY_CACHE.sum() as u64;
+        if ec_usage > GIB || ec_usage > usage / 3 {
```

Comment:
I think most of the time it will trigger the first condition. Is it GIB too small?